Makefile

```diff
@@ -8,6 +8,14 @@ build-all:
 	$(MAKE) -C erc721 build
 	$(MAKE) -C compound-v2 build
 
-.PHONY: erc20-price-spkg
-erc20-price-spkg:
-	cd erc20-price && make pack && mv erc20-price-substreams-v0.1.0.spkg ../target
\ No newline at end of file
+.PHONY: example-all
```

Comment:
What are your thoughts on changing this to `run-all` and changing `example` to `run`? I feel like the latter is more self-explanatory

Reply:
Yep sounds good! - Would you like me to change all individual `make example` commands to `make run` also?

Reply:
yes please